### PR TITLE
Added copied code

### PR DESCRIPTION
**Pull request by WalterOkumu**
Added copied code from Tribute Project as per instructions

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  /* Setting a base font size of 10px give us easier rem calculations
+     Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+   */
+  font-size: 10px;
+}
+
+body {
+  /* Native font stack https://getbootstrap.com/docs/4.2/content/reboot/#native-font-stack */
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
+    'Helvetica Neue', Arial, sans-serif;
```
Comment (WalterOkumu) [inline css/style.css:11]: Please write this on one line, that is font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;

File: index.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+<head>
+  <meta charset='UTF-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+  <title>Tribute Page</title>
+  <link rel='stylesheet' type='text/css' href='css/style.css'>
+	<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>
+	<link href='https://fonts.googleapis.com/icon?family=Material+Icons' rel='stylesheet'>
+	<script src='https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js'></script>
+	<script src='js/script.js' type='text/javascript'></script>
```
Comment (WalterOkumu) [inline index.html:12]: Please remove this line, as there is no JavaScript or related file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+      </p>
+      <cite>- Prof. Wangari Muta Maathai</cite>
+    </blockquote>
+    <figure id='img-div'>
+      <img
+        id='image'
+        src='https://www.thoughtco.com/thmb/rDwnnZ0S34PTdzQ26QLA4bDhH8M=/3777x2501/filters:fill(auto,1)/kenyan-activist-wangari-maathai-525194310-58c341ef5f9b58af5c638e4c.jpg'
+        alt='Prof. Wangari Muta Maathai seen standing infront of a rural farm'
+        height = 740px;
+        width = 100%;
```
Comment (WalterOkumu) [inline index.html:33]: Please delete the height and width, there is already a max-width and height defined in css/styles.css

File: index.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+          <strong>
+            1947
+          </strong>
+          - Returned to Ihithe with her mother and two brothers to attend Ihithe Primary School
+        </li>
+        <li>
+          <strong>
+            1951
+          </strong>
+          - Moved o St. Cecilia's Intermediate Primary School & joined <a target = '_blank' href = 'https://en.wikipedia.org/wiki/Legion_of_Mary'>the Legion of Mary</a>
```
Comment (WalterOkumu) [inline index.html:66]: For all a tags, please include "rel=noopener", disown-opener checks if the rel attribute is specified with both the noopener and noreferrer values (or only noopener if all the [targeted browsers](https://webhint.io/docs/user-guide/configuring-webhint/browser-context/) support it) on a and area elements that have target="_blank" and link to other origins. You can read more about it [here](https://webhint.io/docs/user-guide/hints/hint-disown-opener/)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+          <strong>
+            1956
+          </strong>
+          - Completed her studies, was rated first in her class, and was granted admission to the only Catholic high schoool for girls in Kenya, <a target = '_blank' href = 'https://en.wikipedia.org/wiki/Loreto_High_School,_Limuru'>Loreto High School, Limuru</a>.
+        </li>
+        <li>
+          <strong>
+            1960
+          </strong>
+          - Became one of some 300 Kenyans selected to study in the United States under the <a href = 'https://en.wikipedia.org/wiki/The_Kennedy_Airlift' tagret = '_blank'>The Kennedy Airlift</a>
```
Comment (WalterOkumu) [inline index.html:78]: Please correct "tagret" typo to "target"

**Review by WalterOkumu — COMMENTED** [review]
# Changes Requested

Please check my comments below and make requested changes.

Submit for re-review when done.

Thanks :)

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @WalterOkumu  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Polite comments 👍 
- Descriptive comments 🏄 
- Use of inline comments 🏂 
## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- I would suggest the use of emojis . Emojis bring out expressions that text may not. 👌

**Pull request merged**